# MBZIRC distance finder configuration of ROS parameters

subscribers:
    
    input_bbox:
        topic: /distance_finder/bounding_boxes


publishers:

    target_pos:
        topic: /distance_finder/target_pos
        queue_size: 1
        latch: true

